README.md

```diff
@@ -13,6 +13,9 @@ To get set up with evals, follow the [setup instructions below](README.md#Setup)
 - Familiarize yourself with the existing eval templates: [eval-templates.md](docs/eval-templates.md).
 
 #### Writing evals
+
+**Important: Please note that we are currenlty not accepting Evals with custom code, with the exception of modelgraded evals with custom code.**
```

Comment:
We are accepting modelgraded evals with custom modelgraded YAML files, for example they contribute a new `.yaml` file to the registry here: https://github.com/openai/evals/tree/main/evals/registry/modelgraded

Reply:
So to clarify, we will accept PRs that have new `.yaml` and new `.jsonl` (in git-lfs) files, but if they wrote new `.py` files, we unfortunately can't review them right now

Reply:
Okay, I will adapt the PR accordingly.

Reply:
Please re-review @andrew-openai.